tests/std/tests/P2474R2_views_repeat/test.cpp

```diff
@@ -61,10 +82,13 @@ constexpr void test_common(T val, B bound = unreachable_sentinel) {
 
     static_assert(CanSize<R> == bounded);
 
-    same_as<R> auto rng = views::repeat(val, bound);
-    static_assert(noexcept(views::repeat(val, bound)) == is_nothrow_copy_constructible_v<T>); // strengthened
-    static_assert(noexcept(rng | views::drop(1)) == is_nothrow_copy_constructible_v<T>); // strengthened
-    static_assert(noexcept(rng | views::take(1)) == is_nothrow_copy_constructible_v<T>); // strengthened
+    same_as<R> auto rng = views::repeat(move_if_move_only(val), bound);

```

Comment:
There are several potential uses-after-move of `val` in the following, which is dodgy. I think we should at least have a warning here about using this with types for which move isn't equivalent to copy, but it's not worth resetting testing. I'll do so in a followup.

Reply:
I've prepped a change to add such a comment on my "miscellaneous" branch.